slapp/utils/merge_utils.py

```diff
@@ -0,0 +1,186 @@
+from typing import List, Union, Optional, TypedDict
+from pathlib import Path
+import copy
+import jsonlines
+import boto3
+from urllib.parse import urlparse
+import tempfile
+import shutil
+import logging
+import json
+
+from slapp.transfers.utils import read_jsonlines
+
+
+class MergingException(Exception):
+    pass
+
+
+class WorkerAnnotation(TypedDict):
+    workerId: str
+    roiLabel: str
+
+
+class RecordProject(TypedDict):
+    sourceData: str
+    majorityLabel: str
+    workerAnnotations: List[WorkerAnnotation]
+
+
+def compute_majority(labels):
+    """return majority labels or None if tied
+    """
+    ulabels = set(labels)
+    if len(ulabels) not in [1, 2]:
+        raise ValueError("`compute_majority()` expects binary classification "
+                         f"but received {len(ulabels)} labels: {ulabels}")
+    counts = {ulabel: labels.count(ulabel) for ulabel in ulabels}
```

Comment:
This is fine, you could replace it with a [`collections.Counter`](https://docs.python.org/3.8/library/collections.html#collections.Counter) I think and not even need to create `ulabels` I think?

Reply:
I did that at first, and I didn't find it much better. I could have gone either way